### ISSUE TITLE
Many commands are still stored in index.js

### ISSUE THREAD

**Issue by YodaLightsabr** [ThunderBotDevelopment/ThunderBot#5]
All commands should be moved from index.js to separate command files in the /src/commands folder.

**Comment by WholeRoastedDuck** [comment]
It also seems like /src/economy.js could be split up into multiple files.

**Comment by YodaLightsabr** [comment]
> It also seems like /src/economy.js could be split up into multiple files.

Yes, but that’s lower priority due to the fact that economy.js has exclusively economy commands, whereas index.js has a web server, socket.io server, Slash Commands interactions, reaction listeners, etc.
